cmd/termui/main.go

```diff
@@ -46,6 +47,15 @@ VERSION:
 		Value:       "127.0.0.1:8080",
 		Destination: &argsConfig.address,
 	}
+
+	// gatewayAddress defines a flag for setting the address and port for gateway interactions
+	gatewayAddress = cli.StringFlag{
+		Name:        "gateway-address",
+		Usage:       "Address and port number on which the application will try to connect to gateway",
+		Value:       "https://testnet-gateway.multiversx.com",
```

Comment:
no one will pass this parameter and the termui will end up showing misleading information. I guess we need to find another approach here.

What I can figure out now, is to have a new component *in the node* that will execute the code written in the `loadMetricsFromGatewayApi` function periodically. And the gateway API string will be provided in the config.toml file so we can set it at config time.

Reply:
remove this cli parameter and added a separate metric on node bootstrap metrics